helm/logging-operator/templates/deployment.yaml

```diff
@@ -37,7 +37,10 @@ spec:
         {{- else -}}
           - -enable-logging=false
         {{- end }}
-        - -installation-name={{ .Values.loggingOperator.installationName }}
+        - -installation-name={{ .Values.managementCluster.name }}
+        - -installation-provider={{ .Values.managementCluster.provider.kind }}
+        - -installation-region={{ .Values.aws.region }}
```

Comment:
THis is needed to create an IAM client

Reply:
"region" is probably AWS-specific.
Maybe that should be made clear in its name?

Reply:
Well it is not really as it's also set in azure but I need to move where it is defined in config